src/sql/src/func.rs

```diff
@@ -1610,6 +1610,15 @@ lazy_static! {
                 // encoding id for UTF8 (6) is provided, otherwise return 'NULL'.
                 params!(Int64) => sql_impl_func("CASE WHEN $1 = 6 THEN 'UTF8' ELSE NULL END"), 1597;
             },
+            // pg_get_constraintdef gives more info about a constraint with in the `pg_constraint`
+            // view. It currently returns no information as the `pg_constraint` view is empty in
+            // materialize
+            "pg_get_constraintdef" => Scalar {
+                params!(Oid) => {
```

Comment:
Don't need to add a new func variant here, it's ok to just return an error in a closure. See https://github.com/MaterializeInc/materialize/blob/23c4aff5caf3b4a66abaa0f71f73b3f35a45e860/src/sql/src/func.rs#L1339 for an example.

Reply:
TIL about `bail_unsupported`!